sample/src/androidTest/java/com/schibsted/spain/barista/sample/AssertionsTest.java

```diff
@@ -279,4 +281,14 @@ public void checkSameDrawable() throws Exception {
   public void checkDifferentDrawable() throws Exception {
     assertDrawable(R.id.image_view, R.drawable.ic_action_menu);
   }
+
+  @Test
+  public void checkViewHasFocus() throws Exception {
+    assertHasFocus(R.id.edittext_with_focus);
+  }
+
+  @Test
+  public void checkViewHasNotFocus() throws Exception {
```

Comment:
Add these new methods to `methodsByIdLaunchExceptionWhenThePassedIdIsNotAResIdOrAStringId()` test, to check that they give a great error message if the passed int is not an id.

Reply:
Oh, ok!

Reply:
In this case the method doesn't support string res, only id. 
If we don't do any magic, we won't throw any special error like the other cases in that test.

In this case I think Espresso's message will be good enough.

Reply:
You're right @Sloy , but I think I can do this PR better by allowing to access the view not only by `id`, but with a text, e.g for a button!

Give me some time and will add that